File: src/main/java/Second.java
import interpreter.FirstLexer;
import interpreter.SecondLexer;
import interpreter.SecondParser;
import org.antlr.v4.runtime.CharStreams;
import org.antlr.v4.runtime.CommonTokenStream;

public class Second {
    public static void main(String[] args) {
        var lexer = new FirstLexer(CharStreams.fromString(
                "25 + 3 * 2\n" +
                        "42 - (-69)".trim()));
        var parser = new SecondParser(new CommonTokenStream(lexer));
        var tree = parser.start();
        System.out.println(tree.toStringTree());

    }

}

File: src/main/kotlin/Second.kt
import interpreter.SecondLexer
import interpreter.SecondParser
import org.antlr.v4.runtime.CharStreams
import org.antlr.v4.runtime.CommonTokenStream

fun main() {
    val lexer = SecondLexer(CharStreams.fromString("""
        25 + 3 * 2
        42 - (-69)
    """.trimIndent()))
    val a = CommonTokenStream(lexer)
    val parser = SecondParser(a)
    val tree = parser.start()
    println(tree.toStringTree())

}
